snakefile.py

```diff
@@ -218,20 +240,23 @@ def tool(name):
 
 
 rule fastqc_raw:
-    input: os.path.join(READS_DIR, "{sample}_R{read_num}.fastq")
+    input: trim_reads_input
     output:
-        os.path.join(FASTQC_DIR, '{sample}', '{sample}_R{read_num}_fastqc.zip'),
-        os.path.join(FASTQC_DIR, '{sample}', '{sample}_R{read_num}_fastqc.html')
-    log: os.path.join(LOG_DIR, 'fastqc_{sample}_raw_R{read_num}.log')
+        r1_rep = os.path.join(FASTQC_DIR, '{sample}', '{sample}_R1_fastqc.html'),
+        r2_rep = os.path.join(FASTQC_DIR, '{sample}', '{sample}_R2_fastqc.html')
+    log: [os.path.join(LOG_DIR, 'fastqc_{sample}_raw_R1.log'), os.path.join(LOG_DIR, 'fastqc_{sample}_raw_R2.log')]
     params:
         output_dir = os.path.join(FASTQC_DIR, '{sample}')
-    shell: "{FASTQC_EXEC} -o {params.output_dir} {input} >> {log} 2>&1"
+    run:
+        tmp_R1_output = os.path.basename(input[0])[:-9] + '_fastqc.html'
+        tmp_R2_output = os.path.basename(input[1])[:-9] + '_fastqc.html'
+        shell("{FASTQC_EXEC} -o {params.output_dir} {input} >> {log} 2>&1 && cd {params.output_dir} &&\
+              mv {tmp_R1_output} {output.r1_rep} && mv {tmp_R2_output} {output.r2_rep}")
```

Comment:
I'm really not a fan of using `cd` here at all.  Please don't.

Reply:
I was not sure how to do path concatenation here...

Can I do `mv {params.output_dir}//{tmp_R1_output} {params.output_dir}//{output.r1_rep}` ??

Reply:
done